package.json

```diff
@@ -19,9 +19,11 @@
     "@types/react": "^18.0.21",
     "@types/react-dom": "^18.0.6",
     "astro": "^3.2.3",
+    "axios": "^1.6.7",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "react-icons": "^4.12.0",
+    "react-select": "^5.8.0",
```

Comment:
we already have material-ui so can we use those components? https://mui.com/material-ui/react-select?